packages/common/src/TruffleConfig.js

```diff
@@ -122,26 +132,23 @@ function addPublicNetwork(networks, name, networkId) {
 // Note: local networks generally have more varied parameters, so the user can override any network option by passing
 // a customOptions object.
 function addLocalNetwork(networks, name, customOptions) {
+  const nodeUrl = getNodeUrl(name);
   const defaultOptions = {
-    host: "127.0.0.1",
     network_id: "*",
-    port: 9545,
-    gas: gas
+    gas: gas,
+    provider: function(provider = nodeUrl) {
```

Comment:
this is great! this should hopefully fix the `Web3WebsocketClient` on local develop instances as well and should fix [this](https://github.com/UMAprotocol/protocol/pull/1910#discussion_r481818716) issue I was having in the serverless integration testing.